test/e2e/deployimage/plugin_cluster_test.go

```diff
@@ -78,14 +78,6 @@ var _ = Describe("kubebuilder", func() {
 		})
 
```

Comment:
@Eileen-Yu,

That is great work 🥇 

We just need to squash the commits to get this one merged.
Could you please squash them so we can move forward here?